edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_courses.scss

```diff
@@ -548,3 +566,17 @@ section.idash-section#membership
   border-top: 8px solid $gray-l5;
   border-bottom: 8px solid transparent;
 }
+
+div.container>div {
+  border: 0 !important;
+  box-shadow: none !important;
+}
+
+section#instructor-dashboard-content,
+.view-teams div.page-content,
+section#course-info-progress,
+.content-wrapper header.page-header,
+div.content-wrapper.main-container div.container div.home {
+  padding-left: 8px;
+  padding-right: 8px;
+}
```

Comment:
No new line at the end. Please, lets use a plugin to capture the error

Reply:
ready! @diegomillan